libs/langchain/langchain/chains/transform.py

```diff
@@ -17,8 +21,13 @@ class TransformChain(Chain):
     """
 
     input_variables: List[str]
+    """The keys expected by the transform's input dictionary."""
     output_variables: List[str]
+    """The keys returned by the transform's output dictionary."""
     transform: Callable[[Dict[str, str]], Dict[str, str]]
+    """The transform function."""
+    coroutine: Optional[Callable[[Dict[str, Any]], Awaitable[Dict[str, Any]]]] = None
```

Comment:
would it be clearer to call this atransform?

Reply:
maybe! It's called coroutine in tools rn but overall atransform would be more consistent